src/services/api.js

```diff
@@ -1,7 +1,15 @@
 export async function getCategories() {
   // Implemente aqui
+  const url = 'https://api.mercadolibre.com/sites/MLB/categories';
+  const fetchUrl = await fetch(url);
```

Comment:
Oque acha de trocar o nome dessa variável?
```suggestion
  const fetchResponse = await fetch(url);
```

Reply:
Muito boa a sugestão. Bem melhor para entender!